pkgs/scorec.yaml

```diff
@@ -3,9 +3,9 @@ dependencies:
   build: [mpi, parmetis, zoltan]
 
 sources:
-  - url: https://github.com/SCOREC/core.git
-    key: git:e219777edd875d35096dfedd7197300b8e2f3837
-    
+  - url: https://github.com/tjcorona/core.git
```

Comment:
I think the version we were using had the error, and the newest version of SCOREC/core wasn't compatible with other things. The easiest fix was to branch and patch the version we were using.